ci_scripts/build_linux.sh

```diff
@@ -25,6 +25,8 @@ for i in $@; do
   eval $i
 done
 
+echo which python $(which python) and path is $PATH
```

Comment:
Do we need to keep this? If it is for debugging the CI build, comment it out.

Reply:
I removed the line. It's no longer needed; I also have a Linux machine now so I find the standard Python path myself :)